CHANGES/629.feature

```diff
@@ -0,0 +1 @@
+Support Python 3.10, build Linux Alpine images
```

Comment:
I think this could be improved as follows:
```suggestion
Added support for Python 3.10 and started shipping platform-specific wheels with the ``musl`` tag targeting typical Alpine Linux runtimes.
```

Reply:
Thanks